.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
It's better to not include .idea related files in the PRs. People won't like it unless all the team uses Idea.